codegen/core/src/main/java/software/amazon/smithy/python/codegen/ClientGenerator.java

```diff
@@ -443,10 +452,12 @@ async def _handle_attempt(
                                 raise $1T(
                                     "No endpoint_uri found on the operation config."
                                 )
-
+                            endpoint_resolver_parameters = StaticEndpointParams(uri=config.endpoint_uri)
+                            logger.debug("Calling endpoint resolver with parameters: %s", endpoint_resolver_parameters)
```

Comment:
We may need to rationalize this against Alex's other Endpoint work. I hope we don't need to add different log statements for each endpoint resolver.

Reply:
Yeah - I'm planning on doing a small refactor of the endpoint generator so that this code wouldn't be duplicated between them. (IE - the only thing that should be different between different endpoint generators is the parameter classes and how they are constructed)

Reply:
See refactoring: https://github.com/smithy-lang/smithy-python/pull/394/commits/2a4e6f09218b852372074369a374a758b05fbe4a

This should play nicely with the logging above.